js/app/restaurant/restaurant.model.js

```diff
@@ -3,7 +3,7 @@
 ((app) => {
 	const RestaurantModel = () => {
 		const Restaurant = (restaurant) => {
-			const self = this;
+			const self = {};
```

Comment:
I'm not sure `self` is accurate, since you're not referencing `this`. Maybe rename to `model`?

Reply:
yea super good point, thanks